### ISSUE TITLE
Multiple errors in the main branch

### ISSUE THREAD

**Issue by invader43** [YamiDaisuke/SuperMarioClone#32]
![image](https://user-images.githubusercontent.com/85817312/218660139-df241a95-dcf8-4bca-a348-bd8e8d69006e.png)
No idea what these errors are , is it because the project was developed on an older version of the Engine ?

**Comment by YamiDaisuke** [comment]
Yes, most likely, I haven opened the project in a while. I'll try to make some time to update to the latest engine version
